server/src/guest/guest.controller.ts

```diff
@@ -190,4 +190,37 @@ export class GuestController {
 	async remove(@Param('uuid') uuid: string): Promise<DeleteResult> {
 		return this.guestService.delete(uuid);
 	}
+
+	/**
+	 * If no admin exists, make the user with the provided google Id or create one and make them an admin
+	 * @param googleId The google Id for the new admin
+	 * @returns The new admin user's primary data
+	 */
+	@Post('createadmin')
+	async createAdmin(
+		@Body() googleId: {id: string},
+	): Promise<primaryData | { error: string }> {
+		if (await this.adminService.doAnyAdminsExist())
+			return { error: 'There are already admins' };
```

Comment:
I think we'd want to return a 403? We can still give the reason.

[on properly formatting errors in nestjs](https://docs.nestjs.com/exception-filters)